docs/msgs_package.md

```diff
@@ -6,8 +6,8 @@ This is a custom message package for message types that are LUCI specific.
-| luci_messages | LuciJoystick <br/><br/> `int32 forward_back` <br/> `int32 left_right` <br/> `string joystick_zone`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
-|               | LuciJoystickScaling <br/><br/> `int32 forward_back (scaled forward back limit)` <br/> `int32 left_right (scaled left right limit)` <br/> `string joystick_zone`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| luci_messages | LuciJoystick <br/><br/> `int32 forward_back` <br/> `int32 left_right` <br/> `int32 joystick_zone` <br/> `int32 input source`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Also, since the joystick_zone used to be a string we should have a table for that mapping